src/Collection.js

```diff
@@ -110,6 +110,16 @@ class Collection {
    * @return {number}
    */
   size() {
+    process.stderr.write('Collection#size() is deprecated. Use Collection#length instead');
```

Comment:
I don't see much harm supporting both. I would just add the other one and not deprecate this one.